[ADD] l10n: Ecuador fiscal doc

samueljlieber opened the pull request:
Task: [#3164933](https://www.odoo.com/web#id=3164933&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

Targets: 16.0

This PR adds Ecuador fiscal localization documentation to Odoo user docs.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/3485).

samueljlieber commented:
Hi @vbe-odoo 👋 I've created the PR for the l10n EC doc! Thank you for putting together the RST in the google doc 🙂

Throughout the doc I made adjustments to the RST and document layout, the changes are summarized below along with my reasoning:

- Moved the *Module installation* under Configuration heading and removed the steps to install `l10n_ec` modules since they are covered by the `Install` reference on line 23 - similar reasoning in https://github.com/odoo/documentation/pull/2430#discussion_r1039563460
- I did not include the screenshots for the **Advanced Settings** tab for Customer Invoice, Withholding, & Purchase Liquidations because I felt the notes for these steps were descriptive enough, and it helps keep the image count down.
- Minor adjustments to sentence structure.

Overall this is a really great start and the doc ready for your review now that the RST & PR is created. Please let me know your thoughts on the changes 🙂

vbe-odoo commented:
Hello @samueljlieber 
Many thanks for creating the PR and improving the EC documentation shared with you.

I've done a review, and I agree with your changes!
It looks good to me!

I've moved this PR to Open by mistake, if you want you can move it back to draft.

**IMPORTANT NOTE**: But it is good to clarify that the EC localization is not yet released - we are close to it - so if the review keeps moving to another team, please wait for our go-to-go before merging this into the User Doc.

Thanks!

vbe-odoo reviewed (approved):
I agree with your changes and the Documentation Looks Good to me.
cc' @ren-odoo @fvz-odoo

StraubCreative commented:
@vbe-odoo moved back to draft. 
While we're waiting for EC release, I can review it before we open the PR and pass to accounting team for merge 👍

StraubCreative commented:
@samueljlieber there's a number of merge conflicts here.
Can you prioritize solving them and publish the PR asap so accounting team can review?
Then work with accounting and VBE on change requests until merge.
TIA/KR 🙏

samueljlieber commented:
Updates in d81207b:
- rebased to latest 16.0 updates ([finance directory restructure](https://github.com/odoo/documentation/commit/ecbb755471c5527ef4758f19c1e7648c30243122)) to resolve merge conflicts
- fixed build errors

Thank you @StraubCreative & @vbe-odoo for reviewing the changes! Sending to @odoo/accounting-doc-review!

ren-odoo commented:
Hi @odoo/accounting-doc-review 

Can you take a look at this PR please?  We're waiting on this documanation to be published in order to share the oficial release of the Ecuatorian Localization. 

Thank you.

Donapi commented on content/applications/finance/fiscal_localizations/ecuador.rst:
```diff
@@ -0,0 +1,570 @@
+
+You can see the information needed to report, which includes base and tax amounts, which also
+includes the tax code within the parenthesis in order to report it to the SRI.
+
```
extra line to be removed

Donapi commented on content/applications/finance/fiscal_localizations/ecuador.rst:
```diff
@@ -0,0 +1,570 @@
+Financial Reports
+=================
+
+In Ecuador, there are fiscal reports that the company presents to SRI. In Odoo, we have two of the
+main financial reports used by companies. These are the reports 103 and 104.
```
I would add the path to both reports in this section, and only one screenshot to illustrate where to find them would be sufficient.  
![image](https://user-images.githubusercontent.com/85342080/219618281-8987ef98-9030-4bed-947d-25bac932402a.png)
Also, could you please check the path to the reports? the tax report appears in the Statement Reports on my runbot
![image](https://user-images.githubusercontent.com/85342080/219618500-5e926555-8fc0-4208-85b8-b967bfbda6cb.png)

Donapi reviewed (commented):
Hello, thank you for this new page, great work :)
I did a high-level review. Here are some general comments based on what we do from our side; feel free to consider them or not:  
- add more links to existing documentation
- avoid putting too many images, especially screenshots of whole screens. We prefer to add screenshots that focus on parts of the screen that illustrate a specific point of what is described to make it easier to understand.
- we prefer to write in the present tense than using future.

samueljlieber commented:
@Donapi thank you for the suggestions!

Updates made in https://github.com/odoo/documentation/pull/3485/commits/133313c120f921c5ee0c0c30385fe2bc23e46e62:
- added links to documentation for EDI, chart of accounts, and credit notes
- removed an unnecessary chart of accounts image
- squashed @ren-odoo's revisions into one commit

@ren-odoo thank you for making some of the edits, can you help me with these remaining? 🙏

1. [Short introduction instead of a glossary](https://github.com/odoo/documentation/pull/3485#discussion_r1109429657)
2. [Explain where the user has to navigate to for Withholding configuration](https://github.com/odoo/documentation/pull/3485#discussion_r1109535016)
3. [Explain where the user has to navigate to for Purchase Liquidation configuration](https://github.com/odoo/documentation/pull/3485#discussion_r1109535169)
4. [Explain where the user has to navigate to for configuring withholding code](https://github.com/odoo/documentation/pull/3485#discussion_r1109544090)
5. [Confirm report path and take new screenshot that shows where to find both reports](https://github.com/odoo/documentation/pull/3485#discussion_r1109584045)

Thank you so much!

ren-odoo commented:
Hi @Donapi 

All the suggestions and changes requested changes have been addressed.  

Is it possible release this first version and then if additional improvements are dectected we handle them in a new PR. 

Our EC market and the Sales teams are already waiting for the Content, so it will be ideal if we can release it early next week. 

Thank  @samueljlieber for the additional changes.

StraubCreative commented:
Squashed commits after change requests round + rebased on [fade918](https://github.com/odoo/documentation/pull/3485/commits/fade9182b1e9b9315ce1f31ac035de97bed1cf57).

Markup cleaning on [e83dce2](https://github.com/odoo/documentation/pull/3485/commits/e83dce28700f930b2a9661bdf66daa38dc5c0538). Checks pass 👍

ren-odoo commented on content/applications/finance/fiscal_localizations/ecuador.rst:
```diff
@@ -0,0 +1,570 @@
+Introduction
+============
+
+Glossary
```
Hi @Donapi for this section, we have recieved feedback from end user and internal staff that's good and practival to have the glosary from the start, that's why we added here. I just did some adjustments for the abreviations.

samueljlieber commented:
0cf7078 Updates:
- squashed and rebased @ren-odoo commits

Ready for review 🙂

jcs-odoo commented:
Hi @vbe-odoo and @ren-odoo  :)
I see Donatienne's review was a comment (not a change request) and gave a few pieces of advice here and there. So considering that this PR got already a few reviews, that Dona isn't available, and that it's urgent... Let's move forward with this PR ^^
(I haven't checked it)

AntoineVDV commented on content/applications/finance/fiscal_localizations/ecuador.rst:
```diff
@@ -0,0 +1,584 @@
+main financial reports used by companies. These are the reports 103 and 104.
+
+To get these reports go to the :guilabel:`Accounting` app and select :menuselection:`Reporting -->
+Statements Reports --> Tax Report` and then filter by `Tax Report 103` or Tax Report 104`.
```
```suggestion
Statements Reports --> Tax Report` and then filter by `Tax Report 103` or `Tax Report 104`.
```

AntoineVDV reviewed (approved):
@robodoo delegate+

jcs-odoo commented:
@robodoo r+
